ethexe/signer/src/lib.rs

```diff
@@ -17,198 +17,90 @@
 // along with this program. If not, see <https://www.gnu.org/licenses/>.
 
 //! Signer library for ethexe.
-
+//!
+//! The crate defines types and related logic for private keys, public keys types,
+//! cryptographic signatures and ethereum address.
+//!
+//! Cryptographic instrumentary of the crate is based on secp256k1 standard
```

Comment:
mb it makes sense to replace crate secp256k1 with k256:
- alloy also uses k256 (less dependencies to build)
- it's pure rust implementation (secp256k1 uses secp256k1-sys)

Reply:
That's a great note.
Transitioning to the k256 is out of the PR context, but can be done in the other PR.

Reply:
tracking issue - #4443

Reply:
resolved in favor of #4443